Source/WebKit/Shared/Cocoa/CoreIPCWebKitSecureCoding.mm

```diff
@@ -33,46 +33,51 @@
 #import "CoreIPCNSCFObject.h"
 #import <pal/cocoa/DataDetectorsCoreSoftLink.h>
 
-@interface DDScannerResult ()
+@interface NSObject ()
 - (NSDictionary *)_webKitPropertyListData;
```

Comment:
This needs something like this:
+ (NSDictionary *)_webKitPropertyListDataSchema